apps/algolia/lib/algolia/update.ex

```diff
@@ -49,35 +96,38 @@ defmodule Algolia.Update do
       body: request
     }
 
-    opts
-    |> Api.post()
-    |> parse_response()
+    :post |> Api.action(opts) |> parse_response()
   end
 
   defp send_update({:error, error}, _, _) do
     {:error, {:json_error, error}}
   end
 
+  defp response_to_objects(body) do
+    {:ok, json} = Jason.decode(body)
+    Enum.map(json["hits"], fn obj -> %{objectID: obj["objectID"]} end)
+  end
+
   @spec parse_response({:ok, HTTPoison.Response.t()} | {:error, HTTPoison.Error.t()}) ::
           success | error
-  defp parse_response({:ok, %HTTPoison.Response{status_code: 200}}), do: :ok
+  defp parse_response({:ok, %HTTPoison.Response{status_code: 200, body: body}}), do: {:ok, body}
   defp parse_response({:ok, %HTTPoison.Response{} = response}), do: {:error, response}
   defp parse_response({:error, %HTTPoison.Error{} = error}), do: {:error, error}
```

Comment:
Shouldn't this go in the Api module? Seems to be more related to requests in general than specific to updating indexes.

Reply:
I wonder if it was that way because we'd wanted to parse responses differently when we're searching for results vs. here where we're manipulating the data. 🤔